mongo/doc.go

```diff
@@ -79,7 +79,8 @@
 //
 // Client Side Encryption
 //
-// Client-side encryption is a new feature in MongoDB 4.2 that allows specific data fields to be encrypted.
+// Client-side encryption is a new feature in MongoDB 4.2 that allows specific data fields to be encrypted. Using this
```

Comment:
I realized our original documentation doesn't mention the build tag so I added it here. Running the examples without specifying the tag results in a panic.